client/__tests__/e2e/puppeteer.setup.ts

```diff
@@ -5,12 +5,11 @@
  */
 
 import { setDefaultOptions } from "expect-puppeteer";
+import fetch from "puppeteer-fetch";
 import { isDebug, isDev } from "./config";
 import * as ENV_DEFAULT from "../../../environment.default.json";
 import { DATASET_METADATA_RESPONSE } from "../__mocks__/apiMock";
-
 // TODO: uncomment this for breadcrumbs tests
```

Comment:
should we remove this comment, too, now that we always test breadcrumbs?

Thank you!